### DISCUSSION TITLE
Generation of Api-Docs

### DISCUSSION THREAD

**Discussion by DavidJakszta** [jellydn/next-swagger-doc#747] [Q&A]
So for clarification this doesnt actually read my code and generate the docs based on that. It just reads the comment i put in my api routes and generates it based off of that right?

**Comment by DavidJakszta** [comment]
and how would i do stuff like 
`requestBody:

        description: Create a new pet in the store

        content:

          application/json:

            schema:

              $ref: '#/components/schemas/Pet'
`

**Reply by jellydn** [reply]
You should check the the example apps to see the usage. Let me know if that doesn't work. Thanks.
